components/automate-ui/src/app/entities/entities.ts

```diff
@@ -38,6 +38,10 @@ export function pendingState(state: StateWithStatus): boolean {
   );
 }
 
+export function pending(status: EntityStatus): boolean {
```

Comment:
@msorens Has this same code https://github.com/chef/automate/pull/2206/files#diff-181a6e5552666a924667f33ca404b600R41 in two PRs. 🤞 No merge problems.

Reply:
yeah i plan on waiting for those to merge first :+1: